swatch-producer-aws/src/main/java/com/redhat/swatch/aws/service/AwsBillableUsageAggregateConsumer.java

```diff
@@ -130,9 +130,16 @@ public void process(BillableUsageAggregate billableUsageAggregate) {
             billableUsageAggregate.getAggregateKey().getOrgId(),
             billableUsageAggregate.getRemittanceUuids(),
             billableUsageAggregate.getWindowTimestamp(),
-            awsUsageWindow);
+            awsUsageWindow,
+            e);
         emitErrorStatusOnUsage(billableUsageAggregate, BillableUsage.ErrorCode.INACTIVE);
       } else {
+        log.warn(
+            "Subscription not found for for aggregateId={} orgId={} remittanceUUIDs={}",
+            billableUsageAggregate.getAggregateId(),
+            billableUsageAggregate.getAggregateKey().getOrgId(),
+            billableUsageAggregate.getRemittanceUuids(),
+            e);
         emitErrorStatusOnUsage(
```

Comment:
I would move the "log.warn" statement to be done in "emitErrorStatusOnUsage". 
Also, I don't think we need to print again the exception since this is already done in another log trace, and both logs use the same trace_id.

Reply:
@Sgitario I think the current way of handling exception is appropriate. If you observe that there are multiple log.warn and a log.error message preceding `emitErrorStatusOnUsage`. I have the option of removing all of them and implementing a unified log.warn, such as `log.warn("Skipping billable usage with id={} orgId={} remittanceUUIDs={})` in the method `emitErrorStatusOnUsage`. However, this appears to be an inappropriate approach to me since I want log.error for usage not found and also little bit of customization in other logs. Additionally, I eliminated the printing exception in local; however, the line number of the caller method and class are not getting printed. It just prints `com.redhat.swatch.azure.exception.DefaultApiException:` Hence I am keeping the same implementation. Let me know if you have thoughts on this.